### PR TITLE
Add Feature Highlights section to Open page

### PR DESCRIPTION
**Pull request by alex-ketch**
Key highlights are:
- Replaced the top level `Stencila /Open` headline with a "sales-pitch tagline". Felt it was redundant since we have the name in the hero nav. I tried to keep the subheading close to the intent of the original, without being repetitive of the headline. Right now it's a bit wordy, so need to bring out an axe to lop off some of it.
- Changed the tooltip `?` to be outlines instead of filled in, to better bring focus to the primary `Open` document button
- Added the `How can Stencila Open  help?` section

Stencila-Open-draft

### CODE REVIEW

File: director/stencila_open/templates/open/main.html [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
 {% block title %}Open{% endblock %}
 {% block header %}{% include "open/open_header.html" with brand_tagline='/open' navbar_id='navbar_open' display_share=user_owns_conversion %}{% endblock %}
 {% block content %}
     <section class="section">
         <div class="columns">
             <div class="column is-8 is-offset-2">
-                <h1 class="title has-text-centered">Stencila /open</h1>
-                <h2 class="subtitle is-5 has-text-centered">Generate reproducible publication previews, and collaborate with colleagues using other formats.</h2>
+                <h1 class="title is-size-4-mobile has-text-centered">
+                  Reproducible research collaboration &amp; publication, minus&nbsp;all&nbsp;the&nbsp;headache.
```
Comment (nokome) [inline director/stencila_open/templates/open/main.html]: "Reproducible research collaboration & publication, minus the headaches" (remove "all the").

File: director/stencila_open/templates/open/main.html [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+                </h1>
+
+                <h2 class="subtitle is-5 has-text-centered">Generate previews, collaborate across file formats, and
+                  concentrate on getting your work done.
+                  <br />
+                  Need to share your Jupyter notebook with your supervisor as a Word document for&nbsp;sign&#8209;off?&nbsp;Easy&nbsp;peasy.
```
Comment (nokome) [inline director/stencila_open/templates/open/main.html]: To make this shorter I think that "Generate previews, collaborate across file formats, and concentrate on getting your work done." could be removed. But not convinced it's necessary.

I think that using animated typing could be a way to keep the text short while showing the flexibility of the tool. Maybe a loop deleting and typing `["Jupyter Notebook on Github", "R Markdown document", "Markdown document on HackMD"]` etc or  `["Jupyter Notebook", "R Markdown document"]`.

File: director/stencila_open/templates/open/main.html [hunk 3]
```diff
@@ -184,11 +194,109 @@ <h5><strong>Github</strong></h5>
...
+              <img src="{% static 'img/file-conversion-graphic.png' %}" alt="" />
+            </div>
+
+            <div class="column is-4 is-offset-1">
+              <h3 class="title is-size-4-mobile">
+                Seamless file conversion
+              </h3>
+
+              <p class="is-size-5">
+              Go back-and-forth across file formats as needed so you can efficiently collaborate
```
Comment (nokome) [inline director/stencila_open/templates/open/main.html]: "Go" -> "Convert" ?

File: director/stencila_open/templates/open/main.html [hunk 4]
```diff
@@ -184,11 +194,109 @@ <h5><strong>Github</strong></h5>
...
+
+              <hr class="is-small">
+            </div>
+          </div>
+
+
+          <div class="columns is-vcentered is-centered has-background-light">
+            <div class="column is-4 is-offset-1">
+              <h3 class="title is-size-4-mobile">
+                Preserve data across formats
```
Comment (nokome) [inline director/stencila_open/templates/open/main.html]: "data" -> "your code"

File: director/stencila_open/templates/open/main.html [hunk 5]
```diff
@@ -184,11 +194,109 @@ <h5><strong>Github</strong></h5>
...
+              </h3>
+
+              <p class="is-size-5">
+                By embedding the source-code inside generated figures, we are able to maintain reproducibility
```
Comment (nokome) [inline director/stencila_open/templates/open/main.html]: "By embedding your source code inside generated figures and other outputs"

File: director/stencila_open/templates/open/main.html [hunk 6]
```diff
@@ -184,11 +194,109 @@ <h5><strong>Github</strong></h5>
...
+              <h3 class="title is-size-4-mobile">
+                Beautifully styled papers to match&nbsp;your&nbsp;needs
+              </h3>
+
+              <p class="is-size-5">
+                Instantaneously convert your manuscript into a delightfully readable, semantic and easy-to-share web pre-print.
+              </p>
+
+              <p class="is-size-5">
+                The themes, like all our work, are open-source and can be changed based on your needs.
```
Comment (nokome) [inline director/stencila_open/templates/open/main.html]: Add "Create your own themes".

File: director/stencila_open/templates/open/main.html [hunk 7]
```diff
@@ -184,11 +194,109 @@ <h5><strong>Github</strong></h5>
...
+
+
+          <div class="columns is-vcentered is-centered has-background-white-bis">
+            <div class="column is-3">
+              <img src="{% static 'img/file-conversion-themes.png' %}" alt="" />
+            </div>
+
+            <div class="column is-4 is-offset-1">
+              <h3 class="title is-size-4-mobile">
+                Beautifully styled papers to match&nbsp;your&nbsp;needs
```
Comment (beneboy) [inline director/stencila_open/templates/open/main.html]: Personally I cringe every time a company describes their products as "beautiful". I find it extremely overused these days.

File: director/stencila_open/templates/open/main.html (hunk 6 above)
Comment (alex-ketch) [reply director/stencila_open/templates/open/main.html]: I changed the wording here, mind taking another look please @nokome?

File: director/stencila_open/templates/open/main.html (hunk 7 above)
Comment (nokome) [reply director/stencila_open/templates/open/main.html]: I don't feel quite as strongly as Ben but do also hesitate a little with that wording. Plus, I don't really think the generated HTML  documents are always beautiful - it depends a lot on the content.

Comment (alex-ketch) [reply director/stencila_open/templates/open/main.html]: Mind suggesting an alternative please @beneboy?

**Comment by nokome** [comment]
I feel that the examples are now too far down the page. Some options could be:

- make it a section below "Seamless file conversion".
- make the examples a horizontally scrollable carousel below the form (my preferred option; https://creativebulma.net/product/carousel/demo may be of use)

File: director/stencila_open/templates/open/main.html (hunk 7 above)
Comment (beneboy) [reply director/stencila_open/templates/open/main.html]: Don't feel the need to change it just on my account, just my opinion. My first thought was "elegantly styled" but that's in the same vein, like come on… beautiful?, elegant? it's a freakin HTML page!

What about just "Papers style to match your needs"?

File: director/stencila_open/templates/open/main.html [hunk 8]
```diff
@@ -221,19 +219,18 @@ <h3 class="title is-size-4-mobile">
               </h3>
 
               <p class="is-size-5">
-              Go back-and-forth across file formats as needed so you can efficiently collaborate
-              with every stakeholder.
+                Conver back-and-forth across file formats as needed so you can efficiently collaborate with every
```
Comment (nokome) [inline director/stencila_open/templates/open/main.html]: missing t

File: director/stencila_open/templates/open/main.html (hunk 7 above)
Comment (alex-ketch) [reply director/stencila_open/templates/open/main.html]: Thanks for the input everyone, helps guide the changes.
I changed it to `Multiple themes to style your manuscript` to keep it objective and highlight the fact that users have options.

**Comment by alex-ketch** [comment]
I looked into and [fixed](https://github.com/stencila/hub/pull/252/commits/b2221c2edbcc16bcb4a261aa22a642934c24b6dd) the issue we observed during the call yesterday, with custom components not being properly styled when changing the themes.
Was due to a hardcoded Thema version which didn't include the CSS variables. I've changed to be range `<=1` now.

---

> At a minimum I think the "Upload File" button should be solid (like the "Open" button). But I also think we could go a little further to put the upload functionality on more of an equal footing with opening a URL (currently if feels very secondary).

I also think having it all in one line would be the ideal solution, but should be done in another PR. 
Changing the alignment requires moving around some related pieces (tooltips, errors), and potentially changing the form setup, hence why I'd want to be more cautious with the changes.

For now I've made it solid to match the `Open` button. Please feel free to revert @jwijay.

> I think that we can iterate on the rPNG schematic some more to better capture the idea of embedding code in the data. But for now I think we should at least make the sample code more realistic e.g. plot(data) or plot(height~width, data)

I've changed the graphic based on your `plot(height~width, data)` example.

---

There are a few other suggestions made for this PR, but I'd like to explore them as separate PRs if no one objects.

1.
 > I feel that the examples are now too far down the page. Some options could be:
  > - make it a section below "Seamless file conversion".
  > - make the examples a horizontally scrollable carousel below the form (my preferred option; https://creativebulma.net/product/carousel/demo may be of use)

Agreed, but I think @beneboy is making changed to the examples section, so would defer to his PR for placement changes.

2. 
> To make this shorter I think that "Generate previews, collaborate across file formats, and concentrate on getting your work done." could be removed. But not convinced it's necessary.
>
>I think that using animated typing could be a way to keep the text short while showing the flexibility of the tool. Maybe a loop deleting and typing ["Jupyter Notebook on Github", "R Markdown document", "Markdown document on HackMD"] etc or ["Jupyter Notebook", "R Markdown document"].
